components/local-app/cmd/workspace-stop.go

```diff
@@ -34,54 +34,60 @@ var workspaceStopCommand = &cobra.Command{
 			return err
 		}
 
-		slog.Info("stopping workspace...")
+		slog.Debug("stopping workspace...")
```

Comment:
By default, we only output the `waiting for workspace to stop...` which can be found below. We essentially had 2 very similar messages, which are more useful for debugging than actually using the CLI.